packages/grafana-ui/src/components/DataLinks/DataLinksInlineEditor/DataLinksListItem.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { DataLinksListItem, DataLinksListItemProps } from './DataLinksListItem';
+
+function setupTestContext(options: Partial<DataLinksListItemProps>) {
+  const defaults: DataLinksListItemProps = {
+    index: 0,
+    link: { url: '', title: '', onBuildUrl: jest.fn(), onClick: jest.fn() },
+    data: [],
+    onChange: jest.fn(),
+    onEdit: jest.fn(),
+    onRemove: jest.fn(),
+    suggestions: [],
+  };
+
+  const props = { ...defaults, ...options };
+  const { rerender } = render(<DataLinksListItem {...props} />);
+
+  return { rerender, props };
+}
+
+describe('DataLinksListItem', () => {
+  describe('when link has title', () => {
+    it('then the link title should be visible', () => {
+      const link = {
```

Comment:
Wonder if we could define this variable at the top to avoid recreating it in each test, this way we would also be able to modify only certain props when needed, e.g.:
```
const customLink = {
   ...link, 
   title: (undefined as unknown) as string,
};
```

Reply:
Great suggestion!

Reply:
Fixed!